package.json

```diff
@@ -1,12 +1,15 @@
 {
-  "name": "Salesfore_Canvas_Demo",
+  "name": "salesfore-canvas-demo",
   "version": "0.0.1",
   "description": "Salesforce Canvas Demo",
   "main": "app.js",
+  "engines": {
+    "node": "^20.9.0"
+  },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node app.js",
-    "refresh-react-app": "./build_and_copy.sh"
+    "refresh-react-app": "cross-env ./build_and_copy.sh"
```

Comment:
Required as I couldn't run this command on windows. The following error was observed

```
> salesfore-canvas-demo@0.0.1 refresh-react-app
> ./build_and_copy.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
```